xtask/codegen/js.ungram

```diff
@@ -1257,8 +1257,10 @@ JsImportDefaultClause =
 
 // import * as foo from "mod";
 // import type * as foo from "mod";
+// import defer * as foo from "mod";
 JsImportNamespaceClause =
 	'type'?
+	phase: 'defer'?
```

Comment:
@denbezrukov i add an `phase` fields here, i will expand this fileds in the next pr when i support `phase: 'source'`, that case will be different from `import defer` here.

Reply:
Do you mean that a 'source' phase doesn't make any sense for a namespace syntax?
I think you're right!

To be honest I have the same feeling for a 'type' token.
It might be also a 'phase' modifier. 
But I'm not sure how many changes it might require.